src/game/Chat/Chat.cpp

```diff
@@ -1063,6 +1063,7 @@ ChatCommand * ChatHandler::getCommandTable()
         { NODE, "unlearn",        SEC_ADMINISTRATOR,  false, &ChatHandler::HandleUnLearnCommand,             "", nullptr },
         { NODE, "removeriding",   SEC_ADMINISTRATOR,  false, &ChatHandler::HandleRemoveRidingCommand,        "", nullptr },
         { NODE, "distance",       SEC_ADMINISTRATOR,  false, &ChatHandler::HandleGetDistanceCommand,         "", nullptr },
+        { NODE, "angle",          SEC_ADMINISTRATOR,  false, &ChatHandler::HandleGetAngleCommand,            "", nullptr },

```

Comment:
Shouldn't this be in a seperate pr?

Reply:
No, this is irrelevant. Just needed to see my angle to creatures for debugging.